pages/docs/core-concepts/error-handling.mdx

```diff
@@ -104,7 +107,7 @@ Given that `FooError` has a `_tag` field, we can use the built-in `catchTag` met
 // @include: errors-1
 // ---cut---
 const program = pipe(
-//    ^?
+  //    ^?
```

Comment:
Can we remove the leading spaces for all of these instances? Just to make sure we keep things consistent.

Reply:
Unfortunately, this is dependent on prettier, I'm not sure if there's a way to do that. It's aligning the comment with the code underneath it since it's on the next line. Doesn't seem like there's a way I can prevent it from doing that.